src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -41,65 +41,70 @@ def setUp(self):
             addon=self.three_ver.addon, version='123.45.1')
         # no matching versions (edge cases)
         self.over = Block.objects.create(
-            addon=addon_factory(),
+            addon=addon_factory(file_kw={'hash': 'overhash'}),
             updated_by=user_factory(),
             max_version='0')
         self.under = Block.objects.create(
-            addon=addon_factory(),
+            addon=addon_factory(file_kw={'hash': 'underhash'}),
             updated_by=user_factory(),
             min_version='9999')
+        self.over_tuple = (
+            self.over.guid,
+            self.over.addon.current_version.version,
+            self.over.addon.current_version.all_files[0].hash)
+        self.under_tuple = (
+            self.under.guid,
+            self.under.addon.current_version.version,
+            self.under.addon.current_version.all_files[0].hash)
 
     def test_all_guids(self):
         all_guids = get_all_guids()
         assert len(all_guids) == File.objects.count() == 10 + 8
-        assert (self.three_ver.guid, '123.40') in all_guids
-        assert (self.three_ver.guid, '123.5') in all_guids
-        assert (self.three_ver.guid, '123.45.1') in all_guids
-        over_tuple = (self.over.guid, self.over.addon.current_version.version)
-        under_tuple = (
-            self.under.guid, self.under.addon.current_version.version)
-        assert over_tuple in all_guids
-        assert under_tuple in all_guids
+        assert (self.three_ver.guid, '123.40', '') in all_guids
```

Comment:
Why is there no xpi hash in these `assert`?

Reply:
hash wasn't set when the addon/version was created so it's the default (`''`).

Reply:
Can we set more realistic values? Or do we have non-default values in other test cases? (I quickly looked but could not find anything, maybe I missed something)